sdcm/cluster.py

```diff
@@ -3215,6 +3226,10 @@ def query_metadata(self, url: str, headers: dict = None, token_url: str = None,
     def systemd_version(self) -> int:
         return get_systemd_version(self.remoter.run("systemctl --version", ignore_status=True).stdout)
 
+    @property
+    def is_diagnostics_logged(self):
```

Comment:
I believe this diagnostics dump could be useful - shouldn't we collect it with coredump thread (can be part of separate PR)